docs/source-pytorch/advanced/model_parallel/fsdp.rst

```diff
@@ -2,138 +2,439 @@
 
 .. _fully-sharded-training:
 
+###################################################
+Train models with billions of parameters using FSDP
+###################################################
+
+Use Fully Shared Data Parallel (FSDP) to train large models with billions or trillions of parameters efficiently on multiple GPUs and across multiple machines.
```

Comment:
I think trillions (plural) is too wild of a claim. Even if it's theoretically possible, we don't have any example of it.

What about

"FSDP is regularly used to train large models up to a trillion ..."

Reply:
That's why I've updated all titles to include billions already